Source/Public/Test-ADOPSConnection.ps1

```diff
@@ -0,0 +1,7 @@
+function Test-ADOPSConnection {
+    [CmdletBinding()]
```

Comment:
Please add the decorator `[SkipTest('HasOrganizationParameter')]` to this command to make module tests work. (See `GetADOPSDefaultOrganization` for example)

Reply:
Added. Thank you! Learning new pester things everyday! 😎